tests/test_integration.py

```diff
@@ -195,3 +195,24 @@ def test_regular_after_deps_only():
     result = tox("-e", NATIVE_TOXENV, "--print-deps-only", prune=False)
     assert "bin/python" not in result.stdout
     assert "six" in result.stdout
+
+
+def test_print_deps_without_python_command(tmp_path):
+    bin = tmp_path / "bin"
+    bin.mkdir()
+    tox_link = bin / "tox"
+    tox_path = shutil.which("tox")
+    tox_link.symlink_to(tox_path)
+    env = {**os.environ, "PATH": str(bin)}
```

Comment:
Shouldn't this prepend to PATH, rather than replace it?

Reply:
We want a PATH without `python`. But we need the `tox` command to work. Hence we create custom PATH that contains `tox` only. Previous PATH already contains `tox`.